osutil/disks/disks.go

```diff
@@ -72,6 +72,8 @@ type Disk interface {
 	// matching the specified filesystem label on the disk. Note that for
 	// non-ascii labels like "Some label", the label will be encoded using
 	// \x<hex> for potentially non-safe characters like in "Some\x20Label".
+	// If the filesystem label was not found on the disk, and no other errors
+	// were encountered, ErrFilesystemLabelNotFound will be returned.
```

Comment:
see below about naming and then we would have to say "a FilesystemNotFoundError will be returned"